<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminwindow</class>
 <widget class="QMainWindow" name="adminwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Admin Menu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Create a new user</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adminCUser">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adminCPass">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="adminAddUserbttn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add User</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Change a user's password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>New Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_changeUser">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_changePass">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="changePassBttn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="importBttn">
    <property name="geometry">
     <rect>
      <x>278</x>
      <y>300</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Import a file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Create a new admin</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_newAdminUser">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_newAdminPass">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addAdminbttn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Admin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backbttn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>backbttn</sender>
   <signal>clicked()</signal>
   <receiver>adminwindow</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adminAddUserbttn</sender>
   <signal>clicked()</signal>
   <receiver>adminwindow</receiver>
   <slot>on_adminAddUserbttn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addAdminbttn</sender>
   <signal>clicked()</signal>
   <receiver>adminwindow</receiver>
   <slot>on_addAdminbttn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>importBttn</sender>
   <signal>clicked()</signal>
   <receiver>adminwindow</receiver>
   <slot>on_importBttn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>changePassBttn</sender>
   <signal>clicked()</signal>
   <receiver>adminwindow</receiver>
   <slot>on_changePassBttn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_adminAddUserbttn_clicked()</slot>
  <slot>on_changePassBttn_clicked()</slot>
  <slot>on_importBttn_clicked()</slot>
  <slot>on_addAdminbttn_clicked()</slot>
  <slot>on_pushButton_clicked()</slot>
 </slots>
</ui>
